src/compile.ts

```diff
@@ -26,6 +35,10 @@ export function compile(
   _config?: FunctionlessConfig,
   _extras?: TransformerExtras
 ): ts.TransformerFactory<ts.SourceFile> {
+  const ignore = _config?.exclude
+    ? new Set(glob.sync(_config.exclude, { absolute: true }))
```

Comment:
is this the most efficient way to do this? This could be a huge set?

Reply:
Updated to use `minimatch` (used by npm internally) which turns globs into regex, can be cached, and can match a single value.